docs/usage/ipv6.md

```diff
@@ -1,38 +1,22 @@
 # DualStack Support for Gardener GCP Extension
 
-This document provides an overview of DualStack support for the Gardener GCP extension, detailing its functionality, requirements, and implementation specifics. The document also clarifies the differences between provisioning methods and the unique components required for DualStack clusters.
+This document provides an overview of DualStack support for the Gardener GCP extension.
+Furthermore it clarifies which components are utilized, how the infrastructure is setup and how a DualStack cluster can be provisioned.
 
 ---
 
 ## Overview
 
-DualStack support allows Gardener GCP shoot clusters to leverage both IPv4 and IPv6 addresses. It is supported exclusively when using the **InfraFlow Controller**. The legacy Terraform reconciler does not support DualStack provisioning.
+DualStack support allows Gardener GCP shoot clusters to leverage both IPv4 and IPv6 addresses.
 
-### Key Features
+### Key Components
 
-- **DualStack Subnets**: Separate subnets are created for nodes and services, with explicit IPv4 and IPv6 ranges.
-- **Ingress-GCE Component**: Responsible for creating IPv6 Load Balancers.
+- **DualStack Subnets**: Separate subnets are created for nodes and services, with explicit IPv4 and external IPv6 ranges.
```

Comment:
This makes it sound like there is an IPv4 node subnet and an additional IPv6 node subnet, which is not the case as far as I know. There is a special (additional) IPv6 subnet for services, but I would not position it as a key component, i.e. mention it in the next section only.